go/arm/Dockerfile.tmpl

```diff
@@ -32,13 +32,18 @@ RUN apt install -y \
         libsqlite3-0:arm64
 
 # libsystemd-dev
-RUN apt install -y \
+RUN apt install -y --fix-broken \
```

Comment:
From my experience, `--fix-broken` makes things works, the only good way to resolve dependency changes in cross architecture is manually

Reply:
Do you mean to remove it?